styles/style.scss

```diff
@@ -0,0 +1,465 @@
+                        margin-bottom: .5625rem;
+                    }
+                    @media screen and (max-width: 375px) {
+                        margin-bottom: .5rem;
+                    }
+                }   
+                .table-body-cell {
+                    @extend .table-head-cell;
+                    font-size: 1.25rem;
+                    font-weight: 500;
+                    text-align: left;
+                    &:first-child {
+                        font-size: 2.375rem;
+                    }
+                    @media screen and (max-width: 1280px) {
+                        font-size: 1.125rem;
+                        &:first-child {
+                            font-size: 1.875rem;
+                        }
+                    }
+                    @media screen and (max-width: 1024px) {
+                        font-size: 1rem;
+                        &:first-child {
+                            font-size: 1.75rem;
+                        }
+                    }
+                    @media screen and (max-width: 768px) {
+                        font-size: .75rem;
+                        &:first-child {
+                            font-size: 1.25rem;
+                        }
+                    }
+                    @media screen and (max-width: 575px) {
+                        font-size: 1rem;
+                        &:first-child {
+                            font-size: 1.75rem;
+                        }
+                    }
+                    @media screen and (max-width: 480px) {
+                        font-size: .875rem;
+                        &:first-child {
+                            font-size: 1.5rem;
+                        }
+                    }
+                    @media screen and (max-width: 375px) {
+                        font-size: .75rem;
+                        &:first-child {
+                            font-size: 1.25rem;
+                        }
+                    }
+                    &:first-child {
+                        display: flex;
+                        justify-content: center;
+                        align-items: center;
+                        .image {
+                            max-height: 2rem;
+                            &:first-child{
+                                margin-right: .25rem;
+                            }
+                            &:nth-child(2) {
+                                margin-right: .125rem;
+                            }
+                            &:last-child {
+                                margin-right: .0625rem;
+                            }
+                        }
+                        @media screen and (max-width: 1280px) {
+                            .image {
+                                margin-right: .125rem;
+                                max-width: 1.5rem;
+                                max-height: 1.5rem;
+                                img {
```

Comment:
give full height and width in a generic way

Reply:
done